storey/sources.py

```diff
@@ -944,20 +944,12 @@ def _init(self):
                 path,
                 header=0,
                 parse_dates=self._dates_indices,
-                date_parser=self._datetime_from_timestamp,
+                date_format=self._timestamp_format,
```

Comment:
according to [this](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html) date_format is new in 2.0.0, but if a user is running pandas <2.0

Reply:
Thanks!